packages/transformer/src/IModelExporter.ts

```diff
@@ -302,19 +324,19 @@ export class IModelExporter {
       return;
     }
 
-    const opts: ExportChangesOptions
+    const initOpts: ExporterInitOptions
       = typeof accessTokenOrOpts === "object"
-        ? accessTokenOrOpts
+        ? { exportChangesOptions: accessTokenOrOpts }
```

Comment:
why add this wrapping subobject? Can't we just keep the properties inline (using `extends`)?

Reply:
For future proofing.
If there will be a need to have init options for `exportAll` workflow in the future, then we can have separate members for both (`exportAll` and `exportChanges`) workflows. Since these are different workflows, we won't ever need to initialize both of them at the same time.
If we go with the `extends` now, then all the options would be in the same "bucket" and it would be hard to decide which options you actually need when you wanted to call `initialize`.